code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -105,4 +106,4 @@ obj/item/weapon/reagent_containers/hypospray/combat/nanites
 /obj/item/weapon/reagent_containers/hypospray/medipen/morphine
 	name = "morphine medipen"
 	desc = "A rapid way to get you out of a tight situation and fast! You'll feel rather drowsy, though."
-	list_reagents = list("morphine" = 10)
+	list_reagents = list("morphine" = 10)

```

Comment:
Just wondering, but what are you editing with this make GitHub think you changed this?